cabalscan/src/CabalScan/RuleGenerator.hs

```diff
@@ -387,7 +387,7 @@ data UnresolvedCabalDependencies = UnresolvedCabalDependencies
 readCabalFile :: Path b File -> IO Cabal.PackageDescription
 readCabalFile cabalFilePath = do
   let cabalFile = Path.toFilePath cabalFilePath
-  genericPkg <- Cabal.readGenericPackageDescription Verbosity.normal cabalFile
+  genericPkg <- Cabal.readGenericPackageDescription Cabal.silent cabalFile
```

Comment:
Because the meaning of `normal` verbosity apparently changed between the version of cabal included in the snapshots `lts-18..` and `lts-19..`.
See for instance https://github.com/tweag/gazelle_cabal/actions/runs/2575589850 where the standard output is polluted with `Warning...` making it not a valid Json file anymore.